tests/baselines/reference/tsxReactEmitEntities.js

```diff
@@ -12,5 +12,5 @@ declare var React: any;
 
 
 //// [file.js]
-React.createElement("div", null, "Dot goes here: · &notAnEntity; ");
+React.createElement("div", null, "Dot goes here: \u00B7 &notAnEntity; ");

```

Comment:
I'm not entirely sure if it matters, but this behavior is different from Babel's transformer. In theory we want a dot character here instead of a \u sequence.